src/List/ListItem.js

```diff
@@ -355,7 +355,8 @@ class ListItem extends Component {
       style,
     } = this.props;
 
-    const mergedDivStyles = Object.assign({},
+    const mergedDivStyles = Object.assign(
+      {cursor: 'not-allowed'},
```

Comment:
Can this go with the rest of the style declarations? Or does that have some unwanted side effect?